src/main/java/com/hierynomus/mssmb2/dfs/SMB2GetDFSReferralEx.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C)2016 - SMBJ Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hierynomus.mssmb2.dfs;
+
+import com.hierynomus.smbj.common.SMBBuffer;
+
+public class SMB2GetDFSReferralEx {
```

Comment:
In other places, the only JavaDoc you have for things like this is the section in the spec where it is defined.  Is that what you're looking for?
For example:
```
/**
 * [MS-NLMP].pdf 2.2.1.3 AUTHENTICATE_MESSAGE
 */
public class NtlmAuthenticate extends NtlmPacket {
```